bin/as-recon-discover-services

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 quick_scan() {
-  nmap --stylesheet /static/nmap.xsl --open -Pn -T3 --host-timeout 30m -n --system-dns -p- $1 -oA recon/nmap-punched-quick-tcp
+  nmap --privileged --stylesheet /static/nmap.xsl --open -Pn -T3 --host-timeout 30m -n --system-dns -p- $1 -oA recon/nmap-quick-tcp
```

Comment:
Are we assuming that nmap has the linux permissions set via `setcap` or something?

Reply:
Yes. But I am open to better ideas. Maybe we could check if the capability is set? Or just add a config flag saying assume privileged?

Reply:
When I made the tengo scripts that did the nmap scans using a library, I initially used the privileged option. However, I ran into weird issues where scans wouldn't pick up any ports or error out and I found out that it was due to the linux privileges set. I removed the privileges and ran it and it didn't have any issues. This is why I added sudo back into `as-recon-discover-services.tengo`

Reply:
weird. I havent had any issues with this... yet. Do you remember what caused the errors or how to reproduce them?

Reply:
maybe we just remove the `--privileged` and those who want it can use `export NMAP_PRIVILEGED=""` to get it. I'll do some testing locally.